web-admin/src/features/navigation/nav-utils.ts

```diff
@@ -31,6 +31,7 @@ export function isMetricsExplorerPage(page: Page): boolean {
 }
 
 export function isCanvasDashboardPage(page: Page): boolean {
+  // TODO: Change the route to canvas
```

Comment:
Does this require addressing before merge ?

Reply:
No, this can be addressed later in a separate PR. It's more to do with file structuring in the application code